pages/info/info.py

```diff
@@ -7,49 +7,151 @@
+                ),
+                html.Br(),
+                html.H5("Contribution activity time-ranges:"),
+                html.H6("T0---------T1---------T2--------->>"),
+                html.Ul(
+                    [
+                        html.Li("T0: 'now'"),
+                        html.Li("T1: recent past, e.g. 'six months ago'"),
+                        html.Li("T2: further past, e.g. 'eighteen months ago'"),
+                    ]
+                ),
+                html.H5("Contributor activity categories:"),
+                html.Ul(
+                    [
+                        html.Li("Active: last seen between T0 and T1"),
+                        html.Li("Drifting: last seen between T1 and T2"),
+                        html.Li("Away: last seen no more recently than T2"),
+                    ]
+                ),
+            ]
+        ),
+        html.Hr(),
+        html.B("Contributor Consistency"),
+        html.Blockquote(
+            [
+                html.P(
+                    """
+                    It is interesting to know how many contributors are 'fly-by,' creating an issue or a PR only once or a
+                    few times, versus those 'repeat' contributors who make relatively many contributions. The application user can set this threshold
+                    in order to count the population of contributors who fall into either category.
+                    """
+                ),
+                html.Br(),
+                html.H5("Contribution consistency threshold:"),
+                html.Ul(
+                    [
+                        html.Li("N: number of contributions"),
+                    ]
+                ),
+                html.H5("Contributor consistency categories:"),
+                html.Ul(
+                    [
+                        html.Li("Fly-by: # contributions to the project < N"),
+                        html.Li("Repeat: # contributions to the project >= N"),
+                    ]
+                ),
+            ]
+        ),
+        html.Hr(),
+        html.B("Staleness"),
+        html.Blockquote(
+            [
+                html.P(
+                    """
+                    Some contributions have a 'lifespan' between when they are opened and when they are closed.
+                    The 'lifespan' that a time-sensitive contribution to a project has can be very informative. If issues or pull requests are
+                    open for a very long time, for instance, this can signal problems for a community. Only those who are
+                    very familiar with a community can know how 'old' is a negative thing, however, and whether the 'age' of these
+                    contributions is important to consider.
+                    """
+                ),
+                html.P(
+                    """
+                    It should be obvious that this is a close relative to contributor activity states defined above.
```

Comment:
Stating something as "obvious" can be demening. Different wording here or good with taking it out